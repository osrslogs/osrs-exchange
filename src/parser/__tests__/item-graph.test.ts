import { ItemPriceGraph } from '../../types';
import parseItemPriceGraph from '../item-graph';
import graphConfig from '../../util/test-util/graph-mock.json';

describe('parseItemPriceGraph', () => {
  it('it changes the keys correctly', async () => {
    const graph: ItemPriceGraph = parseItemPriceGraph(graphConfig.valid);
    expect(graph).toMatchSnapshot();
  });
});
