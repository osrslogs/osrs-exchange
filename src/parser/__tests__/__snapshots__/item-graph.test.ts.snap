// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parseItemPriceGraph it changes the keys correctly 1`] = `
Object {
  "average": Object {
    "2019-09-05T00:00:00.000Z": 2528557,
    "2019-09-06T00:00:00.000Z": 2527956,
  },
  "daily": Object {
    "2019-09-05T00:00:00.000Z": 2490649,
    "2019-09-06T00:00:00.000Z": 2492999,
  },
}
`;
